frontend/app/work_packages/directives/inplace_editor/inplace-editor-edit-pane-directive.js

```diff
@@ -111,6 +112,9 @@ module.exports = function(
       };
 
       this.isActive = function() {
+        if (EditableFieldsState.forcedEditState) {
+          return false;
+        }
```

Comment:
I have a bad suspicion here:

* I know that there is the requirement to not show the (✓, ✓ ✉, X) symbols when tabbing inside the create view
* I could imagine, that those buttons are displayed whenever the field is "active"
* Now there are two different conclusions from those two facts:
    1. Even in create mode a field may be active, but that does not neccessarily display the afforementioned buttons
    2. Apparently in create mode fields can't be active, luckily that means we will never show the buttons

The second conclusion is an evil one, because one could imagine other things depending on the "activeness" of a field that are valid in create and edit mode. However, I believe the author of the code above has chosen to follow that conclusion.

**tl; dr:** I believe the three lines above follow the following line of thought:
1. in create mode `forcedEditState` state is set (*I agree*)
2. `forcedEditState` means I am in create mode (***I disagree***)
3. in create mode a field can never be active, so that those fancy buttons are never shown (***I disagree***)